docs/DeveloperGuide.adoc

```diff
@@ -721,17 +721,38 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
-
-|`* * *` |user |add a new person |
-
-|`* * *` |user |delete a person |remove entries that I no longer need
-
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
-
-|`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
-
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|`* * *` |user |see my contacts while managing my address book |remember what I need to do without flipping back and forth
+|`* * *` |user |backup the contacts into a xml file |transfer my contact details with another user or load the existing details to another computer of mine
+|`* * *` |user |create new contacts in the address book |add more new contacts in the future
+|`* * *` |user |type commands that do not follow the format |use them without memorizing the formats
+|`* * *` |user |merge duplicate contacts |use my addressbook without clutter
+|`* * *` |user |group my contacts in more than one grouping |search my contacts easily by groups
+|`* * *` |user |filter and search contacts |display important contacts
+|`* * *` |user |update existing contacts from the address book |update existing contacts if they have changed their contact number or details
+|`* * *` |user |add a short description  |recall details about them
+|`* * *` |user |view all my contacts |organise my contacts easily
+|`* * *` |user |group my users |organize my address book
+|`* * *` |user |delete contacts from the address book |remove unwanted contacts
+|`* *` |user |be prompted about upcoming birthdays |prepare a gift for my friends
+|`* *` |user |see an aesthetically pleasing application |use the app daily and not get bored of it so quickly
+|`* *` |user |have ways to enter commands with shortcuts |update and traverse my address book quickly
+|`* *` |user |sync my contacts from my phone |skip adding users manually
+|`* *` |user |star my favourite contacts |quickly access my most important contacts
+|`* *` |user |easily input the commands as the systems learn the behaviour and common commands that I had in the past |auto-correct my words
+|`* *` |user |use link:#natural-language[natural language] to perform actions |use it without memorising the syntax of commands
+|`* *` |user |easily input commands into the link:#command-line-interface[Command Line Interface (CLI)] with the system prompting me with options of each command |easily input commands without having to refer to the user guide
+|`* *` |user |store my contact's birthday |know when to prepare a gift
```

Comment:
Maybe move it above `* * |user |be prompted about upcoming birthdays |prepare a gift for my friends`, since it depends on contacts having a birthday field?